### ISSUE TITLE
No collect data :(

### ISSUE THREAD

**Issue by wmeneses** [fusioninventory/fusioninventory-for-glpi#3454]
Hi, I don't know what the hell I did, but in GLPI it has stopped working, I don't have any php or database errors, the client log says "No Collect job enabled or Collect support disabled server side".

I see it's like I deleted the jobs, now I have no idea what to do. 

I know there is no support for GLPI, but I need help to delete all the settings and leave it default.

**Comment by ddurieux** [comment]
Is the collect module enabled into plugin FusionInventory (or in the form of the agent you check)?

**Comment by wmeneses** [comment]
Yes, I see in "agent modules" the option "Computer inventory" active and "collect data" active, in the dashboard it says "fusioninventory computers : 0".

I don't seem to have any tasks, I have tried to create one but nothing happens.

Is there a way to do a default reset? or do you recommend me to uninstall the plugin and delete the tables by hand from the database only from the plugin?
